addons/im_livechat/models/mail_channel.py

```diff
@@ -87,7 +87,9 @@ def _gc_empty_livechat_sessions(self):
 
     def _execute_command_help_message_extra(self):
         msg = super(MailChannel, self)._execute_command_help_message_extra()
-        return msg + _("Type <b>:shortcut</b> to insert a canned response in your message.<br>")
+        if self.channel_type == 'livechat':
+            return msg + _("Type <b>:shortcut</b> to insert a canned response in your message.<br>")
+        return msg
```

Comment:
I made this livechat specific, because it was strange for odoobot to suggest this interaction when it doesn't work outside of livechat.